src/Microsoft.AspNetCore.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -174,7 +190,9 @@ private IList<ApiParameterDescription> GetParameters(ApiParameterContext context
 
             if (context.ActionDescriptor.BoundProperties != null)
             {
-                foreach (var actionParameter in context.ActionDescriptor.BoundProperties)
+                var controllerBoundPropertyDescriptors = context.ActionDescriptor.BoundProperties
+                    .OfType<ControllerBoundPropertyDescriptor>();
+                foreach (var actionParameter in controllerBoundPropertyDescriptors)
```

Comment:
My comment about `ControllerBoundPropertyDescriptor` was a suggestion. I was hoping the `PropertyInfo` would make the `ModelMetadata` lookup more direct. Since the `ModelMetadataProvider` API doesn't let `PropertyInfo` make anything easier, revert this change.

And, this change should break cases where `BoundProperties` contains `PageBoundPropertyDescriptor `. Those will now be skipped. Why aren't tests failing?